accounts/forms.py

```diff
@@ -0,0 +1,11 @@
+from django.forms import ModelForm
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+
+from django.contrib.auth.models import User
+
+
+class CreateUserForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = ["username", "email", "password1", "password2"]
```

Comment:
..., "password", "password_confirm"] to lepsze nazwy zmiennych.

Reply:
To nie są moje zmienne, to przychodzi z góry, z tego co wiem nie da się tego modyfikować